### ISSUE TITLE
Multiple UI element styling broken with new Rofi version

### ISSUE THREAD

**Issue by aquova** [adi1090x/rofi#62]
With the update to Rofi 1.7.0, a number of elements have been broken, for example all of the 'colorful' themes now have entirely white elements behind each entry. This seems to be a known incompatibility introduced with the latest Rofi version - https://github.com/davatorium/rofi/issues/1397

**Comment by zjeffer** [comment]
I use `launcer.rasi` and `powermenu.rasi`, adding 

```
element-text, element-icon {
    background-color: inherit;
    text-color:       inherit;
}
```

to those files fixes most problems with colors. 

There's still a problem with the spacing of the powermenu: 

![image](https://user-images.githubusercontent.com/4633209/131489707-717c5d36-ba86-40d0-b346-ae716b0c944c.png)

**Comment by stfnwp** [comment]
I also had problems with off-centered icons.
I could fix it by adding
```
    horizontal-align:   0.5;
    vertical-align:     0.5;
```
to `element-icon`

**Comment by Giannie** [comment]
I think that the following is more robust to avoid issues with colour matching:

```
element-text, element-icon {
    background-color: transparent;
    text-color:       inherit;
}
```

**Comment by kazi-sifat-plus9a** [comment]
I also faced another probelm icon and text is not centered 
https://github.com/davatorium/rofi/issues/1419

**Comment by DaveDavenport** [comment]
The icon alignment is a bug that slipped through the cracks.  (I need more people testing git version :hugs: )
That will be fixed  in 1.7.1 I expect to release soon.

**Comment by sTiKyt** [comment]
I just installed latest version of this thing(and rofi, clean system install) and still have problem with white squares behind all text on all colorful themes
